contracts/NFT.sol

```diff
@@ -36,6 +36,17 @@ contract NFT is
         }
     }
 
+    // Overrides IERC6372 functions to make the token & governor timestamp-based
+
+    function clock() public view override returns (uint48) {
+        return uint48(block.timestamp);
+    }
+
+    // solhint-disable-next-line func-name-mixedcase
+    function CLOCK_MODE() public pure override returns (string memory) {
```

Comment:
Consider using block numbers instead of timestamps, because:
- its cheaper
- its logical and consecutive 
- can be planned ahead
- can be used in any type of consensus like Aura, POS, POW